server/src/metadata/field-metadata/field-metadata.entity.ts

```diff
@@ -7,36 +9,55 @@ import {
   PrimaryGeneratedColumn,
   UpdateDateColumn,
 } from 'typeorm';
+import { Authorize, IDField, PagingStrategies, QueryOptions, Relation } from '@ptc-org/nestjs-query-graphql';
 
 import { ObjectMetadata } from 'src/metadata/object-metadata/object-metadata.entity';
 
 export type FieldMetadataTargetColumnMap = {
   [key: string]: string;
 };
 
+@ObjectType('field')
```

Comment:
Pretty much everything can be done with annotations, this might make the model file a bit harder to read. We can improve that by separating entity and dto in 2 different files, also Authorize can live in a dedicated class and queryOptions can probably be exported to be shared in our objects when necessary